src/Command/Config/ImportCommand.php

```diff
@@ -127,8 +130,9 @@ private function configImport(DrupalStyle $io, StorageComparer $storage_comparer
             $io->success($this->trans('commands.config.import.messages.already-imported'));
         } else {
             try {
-                $config_importer->import();
                 $io->info($this->trans('commands.config.import.messages.importing'));
+                $config_importer->import();
+                return TRUE;
```

Comment:
The `execute()` function is expected to return an exit code (1 on failure, 0 or nothing on success), while the private `configImport()` function is expected to return a success boolean (TRUE on success, FALSE or nothing on failure). So it's weird, but consistent with the current usage...